Syntaxes/HTML (Django).tmLanguage

```diff
@@ -1,1204 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Along with this, you can also remove `HTML (Django).JSON-tmLanguage`. It was the source file that this file was generated from.